app/controllers/v5/measurements_controller.rb

```diff
@@ -5,7 +5,8 @@ class MeasurementsController < V5::BaseUserController
 
     def index
       load_measurements
-      render json: @measurements, each_serializer: V5::MeasurementSerializer
+      render json: @measurements, scope: { locale: params[:locale], ministry: ministry },
```

Comment:
@soberstadt Scope didn't have locale or ministry_id, was passing nils to the localized_name and description which defaults to 'en'. Also, ministry_id was a uuid and getting used in Ministry.find(uuid) which converted it to an int and looked by id.